submissions/misshatsker/tiny-js-world/index.js

```diff
@@ -0,0 +1,57 @@
+// ======== OBJECTS DEFINITIONS ========
+
+const human = {
+   species: 'human',
+   legs: 2,
+   hands: 2
+};
+
+const animal = {
+   species: 'animal',
+   legs: 4,
+   hands: 0
+};
+
+const woman = {
+   ...human,
+   name: 'Lilu',
+   gender: 'female',
+   saying: 'Aloha!'
+};
+
+const man = {
+   ...human,
+   name: 'Bob',
+   gender: 'male',
+   saying: 'Have a nice day!'
+};
+
+const dog = {
+   ...animal,
+   name: 'Mira',
+   gender: 'female',
+   saying: 'Woof-woof!'
+};
+
+const cat = {
+   ...animal,
+   name: 'Tyson',
+   gender: 'male',
+   saying: 'Meow-meow!'
+};
+
+const inhabitants = [
+   woman,
+   man,
+   dog,
+   cat
+];
+
+function getInhabitantInfo(inhabitant) {
+   return `${inhabitant.species}; ${inhabitant.name}; ${inhabitant.gender}; ${inhabitant.legs}; ${inhabitant.hands}; ${inhabitant.saying}`;
```

Comment:
This is a pattern here, the code is repeated. Imagine a situation where you would have more properties and you had a task to change the separator from `;` to, let's say, `,`. Find a way to specify keys by hand and then cycle through them getting information from objects. Make sure to **NOT** use `Object.values` and similar methods as they aren't reliable in terms of key order and presence.

Reply:
Thanks for the review! I've improved it, Could you please check it again?